daily_brief/utils/sanitize_output.py

```
@@ -0,0 +1,7 @@
+import re
+
+def sanitize_output(input: str) -> str:
```

Ruff (0.15.7)

[error] Function argument `input` is shadowing a Python builtin

(A002)